variants/file_export.py

```diff
@@ -26,9 +26,11 @@
     VariantScoresFactory,
     annotate_with_joint_scores,
     annotate_with_pathogenicity_scores,
+    annotate_with_face_scores,
```

Comment:
`_face_` to `_gm_` (won't comment further on that matter.

Reply:
done